lib/verify/redacted_response.rb

```diff
@@ -0,0 +1,42 @@
+module Verify
+  # Used to report a response from Verify without revealing any Personally
+  # Identifiable Information by replacing any string "values" with the "X"
+  # character. For example:
+  #
+  #   Verify::RedactedResponse.new({"value" => "Secret"}).parameters
+  #   => {"value" => "XXXXXX"}
+  #
+  class RedactedResponse
+    KEYS_TO_REDACT = %w[value postCode].freeze
```

Comment:
What is `value`? Could this do with a comment explaining the expected format from Verify?

Reply:
The two strings in this constant are the hash keys that contain personal information in Verify responses (as seen in the `spec/fixtures/verify` files). What would have made this clearer?

Reply:
I mean `value` is a bit confusing to read. It took some thought to realize that the Verify response returns hashes for its properties, not the values directly.